parsers/RU.py

```diff
@@ -266,12 +268,9 @@ def fetch_exchange(
     logger: Logger = getLogger(__name__),
 ) -> list:
     """Requests the last known power exchange (in MW) between two zones."""
-    if target_datetime:
-        today = arrow.get(target_datetime, "YYYYMMDD")
-    else:
-        today = arrow.utcnow()
+    today = target_datetime if target_datetime else datetime.now(timezone.utc)
 
-    date = today.format("YYYY-MM-DD")
+    date = today.strftime("%Y-%m-%d")
```

Comment:
```suggestion
    date = today.date().isoformat()
```
Slightly cleaner in my opinion.